.pipelines/build-linux.yml

```diff
@@ -17,6 +17,10 @@ steps:
   displayName: Test C++
 - bash: ./.scripts/linux/java.sh
   displayName: Build + test Java
+  env:
```

Comment:
Can you please add a comment describing:
- What these are for?
- What the publish destination is
- How to obtain the creds

Reply:
- these are the credentials used to publish to oss.sonatype.org
- the creds are stored in the build